### ISSUE TITLE
NVDA 2018.4rc1 Reads Random Emoji in Internet Explorer when navigating through page history

### ISSUE THREAD

**Issue by btman16** [nvaccess/nvda#9045]
Hello,

Using Internet Explorer with NVDA 2018.4rc1, I was in Gmail and went back to a prior page with alt+left arrow.
I heard NVDA intermittently say "Black Diamond" when going back.
Steps to reproduce:
1. Open a text editor.
2. Press win+. (period) for the emoji panel.
3. Choose an emoji to insert it into the document.
4. Close the document.
5. Open Gmail into basic HTML view in Internet Explorer.
6. Open a message.
7. Press alt+left arrow.
Actual result: For me, NVDA intermittently says black diamond, and when I pressed alt+right arrow to go forward, NVDA sometimes said "spade suit" or something similar.

The OS is Windows 10 build 17134.

Thank you very much for your time and I look forward to any input you may have.

Sincerely,

Brandon Tyson

**Comment by leonardder** [comment]
In your steps to reproduce, I"m missing your explicit closing of the emoji panel with escape, i.e. after step 3. Does that change anything?

**Comment by btman16** [comment]
Hello,

I pressed escape to close the emoji panel and still noticed the problem.

Is that what you were asking?

Thanks,

Brandon

On 12/11/18, Leonard de Ruijter <notifications@github.com> wrote:
> In your steps to reproduce, I"m missing your explicit closing of the emoji
> panel with escape, i.e. after step 3. Does that change anything?
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/9045#issuecomment-446084594


-- 
“Be what you are. This is the first step towards becoming better than you are.”
– J. C. Hare & A. W. Hare

**Comment by Adriani90** [comment]
@btman16 maybe the emoji dialog was still in focus, this might have been a bug in Windows 10. Can you reproduce this in NVDA 2020.1 RC1 and Windows 10 1909 Update?

**Comment by btman16** [comment]
Hi,

In that configuration, I can't reproduce it right now.

I wonder if it had to do with some custom group policy or something,
as it was on a college machine with many custom policies and
configurations set.

Thanks,

Brandon

On 5/2/20, Adriani90 <notifications@github.com> wrote:
> @btman16 maybe the emoji dialog was still in focus, this might have been a
> bug in Windows 10. Can you reproduce this in NVDA 2020.1 RC1 and Windows 10
> 1909 Update?
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/9045#issuecomment-622997368


-- 
“Be what you are. This is the first step towards becoming better than you are.”
– J. C. Hare & A. W. Hare

**Comment by Adriani90** [comment]
I cannot imagine that group policies could have caused this, maybe rather a customized windows version or something like that. Anyway, let's close this issue for now as works for me and if it appears again we can reopen. Thanks for reporting.
